evaluator/src/test/kotlin/RuleSetTest.kt

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Copyright (C) 2021 Bosch.IO GmbH.
```

Comment:
Nit: There should be no dot at the end.

Reply:
Commit message:

> Since the only relevant licenses are the concluded

Maybe better: "Since for a license choice only the concluded license is relevant"